.travis.yml

```diff
@@ -11,3 +11,12 @@ script:
   - julia -e 'Pkg.clone(pwd()); Pkg.build("TimeSeries"); Pkg.test("TimeSeries", coverage=true)'
 after_success:
   - julia -e 'cd(Pkg.dir("TimeSeries")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder())'
+
+  # Documenter auto-deploy
+  - julia -e 'Pkg.add("Documenter"); using Documenter'
+  - julia -e 'Pkg.add("MarketData"); using MarketData'
+  - julia -e 'Pkg.add("Plots")'
+  - julia -e 'Pkg.add("PyCall")'
+  - julia -e 'ENV["PYTHON"]=""; Pkg.build("PyCall"); using PyCall'
```

Comment:
Why is `using` necessary here? These are all run as separate Julia sessions, so `using PyCall` here won't bring PyCall into the namespace in the next step.

Reply:
Also it's probably better to set environment variables in the Travis config, i.e. `env: PYTHON=""` in the appropriate spot.

Reply:
well, I try to use `using` to trigger compilation; hope
it keeps from timeout: https://travis-ci.org/JuliaStats/TimeSeries.jl/builds/298415239#L1715

Reply:
If precompilation is what's eating up all of that time, try deploying from 0.7 instead of 0.6. I believe that's been fixed on master.

Reply:
hmm, we currently do not have 0.7 in our build matrix.
And I want that Documenter renders all `@repl`, `@example` blocks in stable version, and it will not pop up depwarn in documentation.